examples/e2e/module/user_phone_number_contact.tf

```diff
@@ -0,0 +1,7 @@
+resource "ilert_user_phone_number_contact" "this" {
+  region_code = "DE"
+  target      = "+4915123456789"
```

Comment:
where is this phone number coming from?

Reply:
This is the example phone number for region code `DE` which is also used in the web-ui. It is fetched from `api/regions` if i'm not mistaken.